addons/storyshots/stories/directly_required/Button.js

```diff
@@ -1,28 +0,0 @@
-import PropTypes from 'prop-types';
```

Comment:
@ndelangen some of the files in the `storyshots/stories` are required directly (`require('path/to/file')`) in the `.storybook/config.js`, where as some others are required via `require.context`. I guess the idea is to test both methods.

I think your changes removes that and just requires everything directly. 

As we aren't testing everything right now I wouldn't block on this, but I wanted to point it out.